src/App.tsx

```diff
@@ -1,63 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classnames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { GoodList } from './Components/GoodList/GoodList';
+import { goodsFromServer } from './Components/GoodsFromServer';
+import { SortType } from './types/SortType';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+) {
+  const preparedGoods: string[] = [...goods];
+
+  if (sortField) {
+    let sortFunction;
```

Comment:
You can just sort in cases and break, it doesn't make sense to save the callback